.github/workflows/docker-build-ethermint-debug.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           cp -R rollmint ethermint
           cd ethermint
           rm -rf .git
-          go mod edit -replace=github.com/celestiaorg/rollmint=./rollmint
+          go mod edit -replace=github.com/rollkit/rollkit=./rollmint
```

Comment:
`./rollmint`?

Reply:
It's broken either way. We need to rethink our testing in general.